packages/synapse-react-client/src/components/SynapseTable/SynapseTableCell/JSON/JSONTableCellRenderer.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import {
+  isJSONObjectAllPrimitiveKeys,
+  isJSONPrimitive,
+} from './JSONRendererUtils'
+import { JSONPrimitiveRenderer } from './JSONPrimitiveRenderer'
+import { JSONArrayRenderer } from './JSONArrayRenderer'
+import { JSONObjectRenderer } from './JSONObjectRenderer'
+import { ComplexJSONRenderer } from './ComplexJSONRenderer'
+
+export type JSONTableCellRendererProps = {
+  value: string | null
+}
+
+export default function JSONTableCellRenderer(
+  props: JSONTableCellRendererProps,
+) {
+  let value: unknown = props.value
+  try {
+    if (typeof value === 'string') {
+      value = JSON.parse(value)
+    }
+  } catch (e) {
+    console.warn('JSONTableCellRenderer: failed to parse JSON', e)
+  }
+
+  if (isJSONPrimitive(value) || value === null) {
```

Comment:
`isJSONPrimitive(value)` will be true if `value` is null, but for some reason TypeScript isn't properly narrowing the type, so it incorrectly thinks `value` could still be null at line 34 (because `typeof null -> "object"`).

This is fixed by adding this superfluous check to see if `value === null`

Reply:
Ok